blog.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <title>Adoptadog</title>
+</head>
+
+<body>
+
+  <header class="header">
+    <div class ="header__logo">
+      <img src="images/adoptadog-logo.svg" alt="adoptadog logo">
+
+      <h1>Seattle’s premiere dog adoption agency</h1>
+    </div>
+
+    <div class="header__cart">
+
+      <img src="images/cart-icon.svg" alt="shopping cart icon">
+
+      <p>$1234.56</p>
+
+      <button type="button" name="Checkout">Checkout</button>
+
+    </div>
+
+
+    <nav class="container">
+        <a href="./index.html">Home / </a>
+        <a href="./dogs.html">Dogs / </a>
+      <a href="./blog.html">Blog </a>
+    </nav>
+
+  </header>
+
+  <section class="container">
+
+    <h2>Adopt a Dog Blog</h2>
+
+    <div class="">
```

Comment:
You'll want to create classes to style these blog posts. Would recommend using flex in here as well. So when pushed to mobile, your text will wrap below your image, and on wider screens your text will be beside your image